.github/workflows/project_tests.yml

```diff
@@ -48,8 +48,8 @@ jobs:
           access_token: ${{ github.token }}
 
       - uses: actions/checkout@v2
-      - run: |  # Needed for git diff to work.
-          git fetch origin master --depth 1
+      - run: |  # Needed for git diff to work. (get_changed_files_output)
+          git fetch origin master --unshallow
```

Comment:
This uses an additional 40MB of disk space and 2 seconds wall clock time, which I believe is acceptable

Reply:
can `fetch_depth 0` replace this? 
https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches